### ISSUE TITLE
UserWarning due to Pandas

### ISSUE THREAD

**Issue by tblachow** [joommf/help#13]
I have found the problem at Ubuntu (16): for the micromagnetic model example,
In the place where:
In [25]: md=oc.MinDriver()
            md.drive(system) 2017/11/8 22:15: Calling OOMMF (system/system.mif) ... [3.1s]

The result is as follows:

/home/tblachow/.local/lib/python3.5/site-packages/oommfodt/oommfodt.py:114: UserWarning: Pandas doesn't allow columns to be created via a new attribute name - see https://pandas.pydata.org/pandas-docs/stable/indexing.html#attribute-access
df.units = dict(zip(columns, units))
============================================
This problem is common for all the joommf examples, I suppose.

**Comment by fangohr** [comment]
Hi @tblachow,

I think this is now fixed (in version 0.7.7). Can you try to update the package, please, using

<pre>
conda update -c conda-forge oommfodt
</pre>

You can check which version you have installed using 
<pre>
conda search -c conda-forge oommfodt
</pre>
and then look for the line with a `*` before the version number - this is the version installed.


The conda package for Linux is built, we are still waiting for the OSX and windows files to be created; so on those platforms you may need to wait a few more hours. (One can see the available versions at https://anaconda.org/conda-forge/oommfodt/files, for example). 

Could you give this a go and -- once you have oommfodt 0.7.7 installed -- confirm if the problem has disappeared or is still there? Thank you!

**Comment by tblachow** [comment]
The problem is solved. All works properly at Ubuntu 16, while I must confess, possibly due to my another activity today, I had to install several times matplotlib. Firstly, I managed oommdt without conda but using normal pyhton3 pip activity and the 0.7.7 oommfodt became active. Later, I have installed anacoda3. Secondly, I used newly installed conda to install matplotlib to avoid another problems related to matplot. This is possibly may today's installation activity with GPU oommf.
